app/src/main/java/com/nexenio/bleindoorpositioningdemo/ui/beaconview/radar/BeaconRadarFragment.java

```diff
@@ -39,23 +35,7 @@
 
     public BeaconRadarFragment() {
         super();
-        IBeaconFilter uuidFilter = new IBeaconFilter() {
-
-            private UUID legacyUuid = UUID.fromString("acfd065e-c3c0-11e3-9bbe-1a514932ac01");
-            private UUID indoorPositioningUuid = UUID.fromString("03253fdd-55cb-44c2-a1eb-80c8355f8291");
-
-            @Override
-            public boolean matches(IBeacon beacon) {
-                if (legacyUuid.equals(beacon.getProximityUuid())) {
-                    return true;
-                }
-                if (indoorPositioningUuid.equals(beacon.getProximityUuid())) {
-                    return true;
-                }
-                return false;
-            }
-        };
-        beaconFilters.add(uuidFilter);
+        beaconFilters.add(getUuidFilter());
```

Comment:
This line is in all classes that extend `BeaconViewFragment`, you might as well add the filter there.